packages/dataviews/README.md

```diff
@@ -0,0 +1,23 @@
+# DataViews
+
+Data Views are an improved and reusable UI that deal with collections of things.
+
+## Installation
+
+Install the module
+
+```bash
+npm install @wordpress/dataviews --save
+```
+
+## API
```

Comment:
Instead of this empty API section, can be copy everything that's inside `src/README.md` and remove that one?